daft/dataframe/dataframe.py

```diff
@@ -158,6 +160,45 @@ def _result(self) -> Optional[PartitionSet]:
         else:
             return self._result_cache.value
 
+    def _explain_broadcast(self):
```

Comment:
This seems like dashboard-specific code. Let's define this outside of `dataframe.py` if possible, likely some kind of `Client` exposed by the dashboard module I think.

Reply:
Sure, I can update it to live inside of the `daft/dashboard.py` file.